consensus/src/tests/subscriber_tests.rs

```diff
@@ -43,10 +40,10 @@ pub async fn spawn_node(
     let committee = mock_committee(&keys[..]);
 
     // Create the storages.
-    let consensus_store_path = temp_testdir::TempDir::default();
-    let consensus_store = make_consensus_store(&consensus_store_path);
-    let certificate_store_path = temp_testdir::TempDir::default();
-    let certificate_store = make_certificate_store(&certificate_store_path);
+    let consensus_store_path = tempfile::tempdir().unwrap();
+    let consensus_store = make_consensus_store(consensus_store_path.path());
+    let certificate_store_path = tempfile::tempdir().unwrap();
```

Comment:
Hopefully you should be able to call `test_utils::temp_dir()`

Reply:
fixed